src/main/java/uk/gov/companieshouse/model/validator/EndDateValidator.java

```diff
@@ -10,24 +10,27 @@
 @Component
 public class EndDateValidator {
 
-    public static final String EMPTY_END_DATE = "Unable to validate. End date is empty";
-    public static final String END_DATE_CONTINUED_TRUE = "Unable to validate. End Date can't exist if continued illness is true";
-    public static final String END_DATE_AFTER_CREATED = "Unable to validate. End date is after date of creation";
-    public static final String END_DATE_BEFORE_START_DATE = "Unable to validate. Illness End Date is before Start Date";
-    public static final String WRONG_FORMAT = "Unable to Parse Date. Wrong format";
+    static final String EMPTY_END_DATE = "Unable to validate. End date is empty";
+    static final String END_DATE_CONTINUED_TRUE = "Unable to validate. End Date can't exist if continued illness is true";
+    static final String END_DATE_AFTER_CREATED = "Unable to validate. End date is after date of creation";
+    static final String END_DATE_BEFORE_START_DATE = "Unable to validate. Illness End Date is before Start Date";
+    static final String WRONG_FORMAT = "Unable to Parse Date. Wrong format";
 
-    DateTimeFormatter dtf = DateTimeFormatter.ofPattern("dd/MM/yyyy", Locale.UK);
+    DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy-MM-dd", Locale.UK);
 
     public String validateEndDate(Appeal appeal) {
         String rawStartDate = appeal.getReason().getIllness().getIllnessStart();
         String rawEndDate = appeal.getReason().getIllness().getIllnessEnd();
         boolean continued = appeal.getReason().getIllness().getContinuedIllness();
 
-        if (!continued && rawEndDate == null) {
+        if ((rawEndDate == null || rawEndDate.isEmpty()) && !continued) {
             return EMPTY_END_DATE;
         }
-        if (!rawEndDate.isEmpty() && continued) {
+
+        if (rawEndDate != null && !rawEndDate.isEmpty() && continued ) {
             return END_DATE_CONTINUED_TRUE;
+        } else if (rawEndDate == null) {
+            return null;
         }
 
         try {
```

Comment:
Maybe a small tweak so that there's only one _return null_ statement. As you've said, by this point the first two ifs aren't true so we've either got **continued = false and an end date** or **continued is true and no end date**. We only want to validate if there's an end date so only drop into the try/catch if there's an end date to validate.

 ```suggestion
        }
        
        if (rawEndDate != null) {
            try {
```